skimage/feature/tests/test_texture.py

```diff
@@ -176,6 +176,34 @@ def test_correlation(self):
         np.testing.assert_almost_equal(energy[0, 0], 0.71953255)
         np.testing.assert_almost_equal(energy[1, 0], 0.41176470)
 
+    def test_mean(self):
+        result = graycomatrix(
+            self.image, [1], [0, np.pi / 2], 4, normed=True, symmetric=True
+        )
+        mean = graycoprops(result, 'mean')[0, 0]
+        np.testing.assert_almost_equal(mean, 1.29166667)
+
+    def test_variance(self):
+        result = graycomatrix(
+            self.image, [1], [0, np.pi / 2], 4, normed=True, symmetric=True
+        )
+        variance = graycoprops(result, 'variance')[0, 0]
+        np.testing.assert_almost_equal(variance, 1.03993055)
```

Comment:
Did you take these reference test values from the [source mentioned in the docstring](https://prism.ucalgary.ca/items/8833a1fc-5efb-4b9b-93a6-ac4ff268091c)?

Reply:
I obtained the values doing the operations on a calculator without rounding/truncating values, only at the end I rounded to 8 decimal digits as some of the other tests were using this precision. 
I can update them to reflect the values from this reference, but it seems that they used a 3 decimal digit precision for the GLCM values, so the values diverge a little bit.

- mean: **reference**: 1.292 **current test value**: 1.29166667
- variance: **reference**: 1.039067 **current test value**: 1.03993055
- std: **reference**: not in reference. 1.019346 (computed from variance value) **current test value**: 1.01976985
- entropy: **reference**: 2.0951 **current test value**: 2.09472904

I also noted that for some of the tests a 3 decimal digit precision is used, while for others it is not specified.
Changing the values to the ones from the reference, the tests pass if precision=3 and fail if precision is not specified.

Reply:
Oh, that's very thorough of you. :) Hmm, then let's just add a comment that the reference values were calculated by hand and are close to the original source if precision 3 is used. :)